base/file.jl

```diff
@@ -48,10 +46,12 @@ julia> pwd()
 ```
 """
 function pwd()
-    b = Vector{UInt8}(undef, 1024)
-    len = RefValue{Csize_t}(length(b))
-    uv_error(:getcwd, ccall(:uv_cwd, Cint, (Ptr{UInt8}, Ptr{Csize_t}), b, len))
-    String(b[1:len[]])
+    path_max = 1024
+    buf = Base.StringVector(path_max)
+    sz = RefValue{Csize_t}(path_max + 1)
```

Comment:
Same problem as in #31434.

Reply:
Are you looking at an outdated version of this PR?

Reply:
Whoops, yes.